ui/app/src/context/DarkMode.tsx

```diff
@@ -24,7 +24,7 @@ const DARK_MODE_PREFERENCE_KEY = 'PERSES_ENABLE_DARK_MODE';
 
 interface DarkModeContext {
   isDarkModeEnabled: boolean;
-  setDarkMode: (pref: boolean) => Promise<void>;
+  setDarkMode: (pref: boolean) => void;
```

Comment:
Oh man, I did this in a rush so long ago 🙃
I was bringing over a bunch of code from an existing implementation that stored the theme preference in the db, thanks for fixing